<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>406</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Comic Sans MS</family>
    <pointsize>11</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:qlineargradient(spread:pad, x1:0.411, y1:0.409, x2:1, y2:1, stop:0.278947 rgba(189, 216, 80, 255), stop:1 rgba(255, 242, 53, 255))</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>631</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>JetBrains Mono ExtraBold</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: white 10px</string>
    </property>
    <property name="text">
     <string>Language Translator Tool</string>
    </property>
   </widget>
   <widget class="QComboBox" name="dropDown">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>310</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox#dropDown{
	background-color:white;
	border-radius:20px;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>310</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btn{
	background-color: rgb(255, 85, 127);
	border-radius: 15px;
	border-width:5px;
	font-size 10px:

}</string>
    </property>
    <property name="text">
     <string>Change!</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textBox2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>140</y>
      <width>321</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit#textBox2 {
	background-color: white;
    border-style: outset;
    border-width: 2px;
    border-radius: 20px;
    border-color: beige;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
}</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="textBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>321</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPlainTextEdit#textBox {
	background-color: white;
    border-style: outset;
    border-width: 2px;
    border-radius: 20px;
    border-color: beige;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
}</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
